svcomp/bench/toInstallDir/runSMACKBench.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
The name of this folder has been changed to `src`.  I'm not sure if this is ideal, since nothing in this folder gets compiled.  However, I couldn't really think of a better name - this folder contains some scripts, some library files for the scripts, and some "overlay" files (which get copied on top of an installation of BenchExec, for example).  With this said, however, everything in the now-named `src` folder only works when they've been copied to an "install" directory, where BenchExec, cpachecker, and sv-comp benchmarks have been installed.